package.json

```diff
@@ -47,6 +47,7 @@
   },
   "keywords": [],
   "dependencies": {
+    "express": "^4.11.2",
```

Comment:
A bit surprissed... was express removed in a previous commit?

Reply:
I did this error when merging changes of master branch... :(

Reply:
NTC